tsl/src/fdw/scan_plan.c

```diff
@@ -228,7 +228,9 @@ evaluate_stable_function(Oid funcid, Oid result_type, int32 result_typmod, Oid r
 		else
 			has_nonconst_input = true;
 	}
-
+	if (has_null_input)
+	{
+	}
```

Comment:
```suggestion
```

Reply:
Fixed by removing the change.

Reply:
I guess you forgot to remove it, no?

Reply:
Right, i had this change in my main branch. Thus this change was re-appearing again and again. Reverted this change in main branch and fixed in current branch as well.